descqa/shear_test.py

```diff
@@ -224,17 +233,25 @@ def post_process_plot(self, ax):
         '''
         Post-processing routines on plot
         '''
+        zmeans = np.linspace(self.zlo, self.zhi, self.ntomo+2)[1:-1]
+
         # vmin and vmax are very rough DES-like limits (maximum and minimum scales)
```

Comment:
Do you mean the DES cuts? Yes, very likely. I went with the convenience of being able to choose the number of bins rather than using set bins with exact DES limits.